rasa/cli/interactive.py

```diff
@@ -55,70 +54,50 @@ def interactive(args: argparse.Namespace):
     _set_not_required_args(args)
 
     if args.model is None:
-        check_training_data(args)
-        zipped_model = train.train(args)
+        zipped_model = train.train_core(args) if args.core_only else train.train(args)
+        if not zipped_model:
+            utils.print_error_and_exit(
+                "Could not train an initial model. Either pass paths "
+                "to the relevant training files (`--data`, `--config`, `--domain`), "
+                "or use 'rasa train' to train a model."
+            )
     else:
         zipped_model = get_provided_model(args.model)
+        if not (zipped_model and os.path.exists(zipped_model)):
+            utils.print_error_and_exit(
+                f"Interactive learning process cannot be started as no initial model was "
+                f"found at path '{args.model}'.  Use 'rasa train' to train a model."
+            )
 
     perform_interactive_learning(args, zipped_model)
 
 
 def _set_not_required_args(args: argparse.Namespace) -> None:
     args.fixed_model_name = None
     args.store_uncompressed = False
-
-
-def interactive_core(args: argparse.Namespace):
-    _set_not_required_args(args)
-
-    if args.model is None:
-        zipped_model = train.train_core(args)
-    else:
-        zipped_model = get_provided_model(args.model)
-
-    perform_interactive_learning(args, zipped_model)
+    args.core_only = False
```

Comment:
this will overwrite potential `core_only=True`s

Reply:
however, it works nevertheless, so you can just remove it :-D

Reply:
Keeping it but setting it correctly -- the idea is if you don't pass a trained model, it will only train core and not NLU